apps/dashboard/src/error-analysis/__mock_data__/dummyTreeWine.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IErrorAnalysisTreeNode } from "@responsible-ai/core-ui";
```

Comment:
Some of these test files are quite large (like the one below with 4k lines). I wonder if we can move them to a zip folder on blob storage at some point. Such large autogenerated test data doesn't really belong in git I think.  Also it makes prettier extensions really slow.

Reply:
opened #1192 for this.